indico/modules/events/abstracts/templates/abstract/public.html

```diff
@@ -232,6 +232,46 @@
         </div>
 {% endmacro %}
 
+{% macro _render_judgment_action(abstract) %}
+    {% set action = abstract.state.name %}
+    {% if abstract.can_judge(session.user) %}
+        {% set judge_name = abstract.judge.name %}
+        {% if action == 'withdrawn' %}
+            {% trans name=judge_name -%}
+                <strong>{{ name }}</strong> withdrew this abstract
```

Comment:
hmm in then maybe we should simply use the judge/judge_id for it (in that case the `judge_if_judged` constraint and the `JUDGED_STATES` set in the migration script need to be updated)